tcpclient/connection.go

```diff
@@ -34,7 +48,7 @@ func (c Connection) String() string {
 	case connectionDialing:
 		status = "dialing"
 	case connectionEstablished:
-		status = "established"
+		status = "established in " + c.timeToEstablished().String()
```

Comment:
i'm not sure if that should be part of the standard connection representation (or part of a different "debug" line). And, in any case, variable "status" is being distorted. I almost prefer a switch (in case the connection has the metric X, print this properly... ). In fact, maybe you should think about defining the method String for the ConnectionMetrics struct...

Reply:
👍

Reply:
Probably it's too soon for a specific String method because I haven't clearly defined the other metrics, so I'm going for a short-term solution :)